botathon/index.php

```diff
@@ -582,91 +582,104 @@ class="rd-parallax-layer section-top-75 section-md-top-150 section-lg-top-260">
 footer(false);
 ?>
 <script>
-	// var currentNav = "#nav-info"
-	// var currentAnchor = '#info'
-	var anchors = ['info','register','schedule','parts-list','teams','sponsors','contacts']
-	var curIndex = 0
-	var lastScrollTop = $(window).scrollTop()
+	let anchorSelectors = [$("#info"), $("#register"), $("#schedule"), $("#parts-list"), $("#teams"), $("#sponsors"), $("#contacts"),]
+	let navSelectors = [document.querySelector("#nav-info"), document.querySelector("#nav-register"), document.querySelector("#nav-schedule"), document.querySelector("#nav-parts-list"), document.querySelector("#nav-teams"), document.querySelector("#nav-sponsors"), document.querySelector("#nav-contacts"),]
+	let footerSelector = $('footer')
+	let curIndex = 0
+	let windowSelector = $(window)
+	let lastScrollTop = windowSelector.scrollTop()
 	const downTolerance = 0.25
 	const upTolerance = 0.5
+	$(window).scroll(navChange);
 
-	$(window).scroll(function(){
-		windoo = $(this)
-		let currentScrollTop = windoo.scrollTop()
-		let windowHeight = windoo.height()
-		let scrollBottom = currentScrollTop + windowHeight
-		function elementInView(elem)
-		{
-			let elemTop = $(elem).offset().top + parseInt($(elem).css('padding-top'),10)
-			var elemBottom = elemTop + $(elem).height()
-			return (elemTop <= currentScrollTop && elemBottom >= currentScrollTop) || (elemTop >= currentScrollTop && elemTop <= scrollBottom) || (elemBottom >= currentScrollTop && elemBottom <= scrollBottom)
-		}
-		function getTop(e){return $(e).offset().top + parseInt($(e).css('padding-top'),10)}
-		function getBottom(e) {return getTop(e)+ $(e).height()}
-		function switchActive(e,i){
-			$('#nav-'.concat(anchors[curIndex])).removeClass('active')
-			$('#nav-'.concat(e)).addClass('active')
-			curIndex = i
+    $(document).ready(function () {
+        // Add smooth scrolling to all links
+        $("#botathon-navigation ul li a").on('click', function (event) {
+
+            // Make sure this.hash has a value before overriding default behavior
+            if (this.hash !== "") {
+                // Prevent default anchor click behavior
+                event.preventDefault();
+
+                // Store hash
+                var hash = this.hash;
+
+                // Using jQuery's animate() method to add smooth page scroll
+                // The optional number (800) specifies the number of milliseconds it takes to scroll to the specified area
+                $('html, body').animate({
+                    scrollTop: $(hash).offset().top
+                }, 800, function () {
+
+                    // Add hash (#) to URL when done scrolling (default click behavior)
+                    window.location.hash = hash;
+                });
+            } // End if
+        });
```

Comment:
The spacing here is really weird and is making it hard to read

Reply:
I think 4f5207c reformats it?